symfony/plugins/orangehrmCorePlugin/test/components/LabelCellTest.php

```diff
@@ -32,7 +32,7 @@ protected function tearDown() {
 
     public function test__toString() {
         $dataObject = new LabelCellTestDataObject();
-        $dataObject->getName = create_function('', "return 'Kayla Abbey';");
+        $dataObject->getName = 'Kayla Abbey';
```

Comment:
shouldn't  this be `$dataObject->getName = function() {
	return 'Kayla Abbey';
};`

Reply:
I thought the same .. but actually not sure why they create that function inside the tests

Reply:
yeah, it seems function is not used, even though defined.